### PR TITLE
Dev member

### PR DESCRIPTION
**Pull request by tuanhai2712**

### CODE REVIEW

File: app/Http/Controllers/MemberController.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            $fileNameImage = mt_rand(1000, 10000).'-'.$fileNameImage;
+        }
+        $pathSaveImage = $request->member_avatar->storeAs('avatar', $fileNameImage);
+        $dataNewMember = $request->all();
+        $profileNewMember = new Member();
+
+        if ($id !== null) {
+            $profileNewMember = Member::find($id);
+        }
+            $profileNewMember->member_name          = $dataNewMember['member_name'];
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberController.php]: Pls run phpcs for this file.

File: app/Member.php [hunk 2]
```diff
@@ -6,9 +6,14 @@
 
 class Member extends Model
 {
-    protected $fillable = 
-    ['member_name', 'member_info', 
-    'member_phone', 'member_dateofbirth', 
+    protected $fillable =
+    [
```
Comment (TaHieu-scuti) [inline app/Member.php]: indentation wrong. Pls run phpcs.

File: resources/views/add.blade.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                @foreach($errors->all() as $error)
+                    <p class="alert alert-danger">{{$error}}</p>
+                @endforeach
+            @endif
+            @if(session('success'))
+                <p class="alert alert-success">{{session('success')}}</p>
+            @endif
+        	<form method="post" enctype="multipart/form-data">
+            	<div class="form-group">
+                	<label>Name</label>
```
Comment (TaHieu-scuti) [inline resources/views/add.blade.php]: Indentation wrong. Pls fix this.

File: resources/views/add.blade.php [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                    <input required type="text" name="member_name" class="form-control" placeholder="Username" />
+                </div>
+                <div class="form-group">
+                	<label>Info</label>
```
Comment (TaHieu-scuti) [inline resources/views/add.blade.php]: The same above.

File: resources/views/admin.blade.php [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+</style>
+</head>
+<body>
+<div class="container">
+    <div id="navbar" class="row">
+      <div class="col-sm-12">
+          <nav class="navbar navbar-default">
+              <div class="container-fluid">
+                  <ul class="nav navbar-nav">
+                    <li><a href="">Home</a></li>
```
Comment (TaHieu-scuti) [inline resources/views/admin.blade.php]: Indentation wrong. Plz fix this.

File: resources/views/edit.blade.php [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+	<div class="row">
+    	<div id="view" class="col-xs-4">
+            @if($errors->any())
+                @foreach($errors->all() as $error)
+                    <p class="alert alert-danger">{{$error}}</p>
+                @endforeach
+            @endif
+        	<form method="post" enctype="multipart/form-data">
+            	<div class="form-group">
+                	<label>Name</label>
```
Comment (TaHieu-scuti) [inline resources/views/edit.blade.php]: Indentation wrong. Pls fix this.

File: resources/views/edit.blade.php [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+                    <input required type="text" name="member_name" class="form-control" value="{{$item['member_name']}}" />
+                </div>
+                <div class="form-group">
+                	<label>Info</label>
```
Comment (TaHieu-scuti) [inline resources/views/edit.blade.php]: The same above.

File: app/Http/Controllers/MemberController.php [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+            $fileNameImage = mt_rand(1000, 10000).'-'.$fileNameImage;
+        }
+        $pathSaveImage = $request->member_avatar->storeAs('avatar', $fileNameImage);
+        $dataNewMember = $request->all();
+        $profileNewMember = new Member();
+
+        if ($id !== null) {
+            $profileNewMember = Member::find($id);
+        }
+            $profileNewMember->member_name = $dataNewMember['member_name'];
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/MemberController.php]: indentation wrong. Pls this file.

File: app/Http/Requests/ValidatorRequest.php [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+
```
Comment (ngon12) [inline app/Http/Requests/ValidatorRequest.php]: Remove this line, Thanks

File: database/migrations/2018_01_17_070722_create_member_project_table.php [hunk 10]
```diff
@@ -17,6 +17,7 @@ public function up()
             $table->increments('id');
             $table->string('member_id');
             $table->string('project_id');
+            $table->timestamps();
```
Comment (ngon12) [inline database/migrations/2018_01_17_070722_create_member_project_table.php]: Please add one more migration file for this, never modify old migration files.

File: resources/views/add.blade.php [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+                        </div>
+                        <button type="submit" name="submit" class="btn btn-primary">Add</button>
+                        {{csrf_field()}}
+                    </form>
+                </div>
+            </div>
+        </div>
+    </body>
+
+    </html>
```
Comment (ngon12) [inline resources/views/add.blade.php]: please add one more line

File: resources/views/admin.blade.php [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+                        <td width="5%" align="center"><a href="{{ route('deletemember', [" id "=>$key['id']])}}">Delete</a></td>
+                    </tr>
+                    @endforeach
+                </table>
+            </div>
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (ngon12) [inline resources/views/admin.blade.php]: Please add one more like to here, thanks

File: resources/views/edit.blade.php [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+                        </div>
+                        <button type="submit" name="submit" class="btn btn-primary">Edit</button>
+                        {{csrf_field()}}
+                    </form>
+                </div>
+            </div>
+        </div>
+    </body>
+
+    </html>
```
Comment (ngon12) [inline resources/views/edit.blade.php]: Please add one more line to here, thanks

File: routes/api.php [hunk 14]
```diff
@@ -15,4 +14,4 @@
 
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
-});
+});
```
Comment (ngon12) [inline routes/api.php]: Add one more line here too, thanks

File: database/migrations/2018_01_17_070722_create_member_project_table.php (hunk 10 above)
Comment (tuanhai2712) [reply database/migrations/2018_01_17_070722_create_member_project_table.php]: migration file name: update_column_table_member_project
public function up()
{
    Schema::table('member_project', function(Blueprint $table)
    {
        $table->timestamp();
    });
}

public function down()
{
    Schema::table('member_project', function(Blueprint $table)
    {
        $table->dropColumn('created_at', 'updated_at');
    });
}
}

Comment (ngon12) [reply database/migrations/2018_01_17_070722_create_member_project_table.php]: ok

**Pull request merged**